docs/api-reference/self-host/usage.mdx

```diff
@@ -1,126 +1,175 @@
+| `getApi`                       | Get the tKey API                                                                    | `void`                                                                                         | `ITKeyApi`               |
+
+</TabItem>
+
+<TabItem value="class">
+
+```ts
+class ThresholdKey implements ITKey {
+  modules: ModuleMap;
+  enableLogging: boolean;
+  serviceProvider: IServiceProvider;
+  storageLayer: IStorageLayer;
+  shares: ShareStorePolyIDShareIndexMap;
+  privKey: BN;
+  lastFetchedCloudMetadata: Metadata;
+  metadata: Metadata;
+  manualSync: boolean;
+  private setModuleReferences;
+  private initializeModules;
+  storeDeviceShare: (deviceShareStore: ShareStore, customDeviceInfo?: StringifiedType) => Promise<void>; // Call to trigger module to store shares
+  constructor(args?: TKeyArgs);
+  static fromJSON(value: StringifiedType, args: TKeyArgs): Promise<ThresholdKey>;
+  getStorageLayer(): IStorageLayer;
+  initialize(params?: {
+    withShare?: ShareStore;
+    importKey?: BN;
+    neverInitializeNewKey?: boolean;
+    transitionMetadata?: Metadata;
+    previouslyFetchedCloudMetadata?: Metadata;
+    previousLocalMetadataTransitions?: LocalMetadataTransitions;
+    delete1OutOf1?: boolean;
+  }): Promise<KeyDetails>;
+
+  reconstructKey(_reconstructKeyMiddleware?: boolean): Promise<ReconstructedKeyResult>;
+  deleteShare(shareIndex: BNString): Promise<DeleteShareResult>;
+  generateNewShare(): Promise<GenerateNewShareResult>;
+  syncLocalMetadataTransitions(): Promise<void>;
+  inputShareStoreSafe(shareStore: ShareStore, autoUpdateMetadata?: boolean): Promise<void>;
+  outputShareStore(shareIndex: BNString, polyID?: string): ShareStore;
+  getKeyDetails(): KeyDetails;
+  addShareDescription(shareIndex: string, description: string, updateMetadata?: boolean): Promise<void>;
+  deleteShareDescription(shareIndex: string, description: string, updateMetadata?: boolean): Promise<void>;
+  updateShareDescription(shareIndex: string, oldDescription: string, newDescription: string, updateMetadata?: boolean): Promise<void>;
+  getTKeyStore(moduleName: string): Promise<TkeyStoreItemType[]>;
+  getTKeyStoreItem(moduleName: string, id: string): Promise<TkeyStoreItemType>;
+  outputShare(shareIndex: BNString, type?: string): Promise<unknown>;
+  inputShare(share: unknown, type?: string): Promise<void>;
+  toJSON(): StringifiedType;
+  getApi(): ITKeyApi;
+
+  _refreshShares(threshold: number, newShareIndexes: Array<string>, previousPolyID: PolynomialID): Promise<RefreshSharesResult>;
+  _initializeNewKey({
+    determinedShare,
+    initializeModules,
+    importedKey,
+    delete1OutOf1,
+  }?: {
+    determinedShare?: BN;
+    initializeModules?: boolean;
+    importedKey?: BN;
+    delete1OutOf1?: boolean;
+  }): Promise<InitializeNewKeyResult>;
+  _setKey(privKey: BN): void;
+
+  {/* reconstructLatestPoly(): Polynomial; */}
```

Comment:
These lines could be uncommented.

Also, I think these are supposed to go at the bottom of the page.